detection/execution/unexpected-execdir-linux.sql

```diff
@@ -87,6 +87,13 @@ WHERE
         cgroup_path LIKE '/user.slice/user-1000.slice/user@1000.service/app.slice/app-gnome-Alacritty-%.scope'
         AND path LIKE '/tmp/%'
       )
+      AND NOT (
+        (
+          cgroup_path LIKE '/user.slice/user-1000.slice/user@1000.service/app.slice/app-org.gnome.Terminal.slice/vte-spawn-%.scope'
+          OR cgroup_path LIKE '/user.slice/user-1000.slice/user@1000.service/user.slice/docker-%.scope'
+        )
+        AND path LIKE '/usr/share/nvm/versions/node/v%/bin/node'
+      )
```

Comment:
Does this do what I think it does: not fire if `cgroup_path` is either of those `AND` when `path` matches?

Reply:
That makes sense to me. If either of the two items are true and the third item is true, then the alert won't fire.